salt/_modules/metalk8s_package_manager.py

```diff
@@ -0,0 +1,78 @@
+'''
+Describes our custom way to deal with yum packages
+so that we can support downgrade in metalk8s
+'''
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'metalk8s_package_manager'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def list_pkg_deps(pkg_name, version=None, fromrepo=None):
+    '''
+    Check dependencies related to the packages installed so that we can pass
+    this information to pkg.installed
+
+    name
+        Name of the package installed
+
+    version
+        Version number of the package
+
+        Use : salt '*' metalk8s_package_manager.list_pkg_deps kubelet 1.11.9
+    '''
+    log.info(
+        'Listing deps for "%s" with version "%s"',
+        str(pkg_name),
+        str(version)
+    )
+    pkgs_list = {pkg_name: version}
```

Comment:
So, `pkgs_list` is actually a dictionary...? Could do with a more explicit name IMHO.